priv/repo/migrations/20240509181234_change_days_of_week_column_type.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Screenplay.Repo.Migrations.ChangeDaysOfWeekColumnType do
+  use Ecto.Migration
+
+  def change do
+    execute(&execute_up/0, &execute_down/0)
+  end
+
+  defp execute_up do
+    repo().query!(
+      "ALTER TABLE pa_message ALTER COLUMN days_of_week TYPE integer[] USING (days_of_week::integer[])"
```

Comment:
**question (non-blocking):** do we want a GIN (or GiST) index on this column?

Reply:
Good idea. Got one added.